pkg/lib/constants/consts.go

```diff
@@ -34,14 +34,15 @@ const (
 	IgnoreFileName = ".kitignore"
 
 	// Constants for the directory structure of kit's cached images and credentials
-	// Modelkits are stored in <configpath>/kitops/storage/ and
-	// credentials are stored in <configpath>/kitops/credentials.json
-	DefaultConfigSubdir = "kitops"
-	StorageSubpath      = "storage"
-	CredentialsSubpath  = "credentials.json"
-	HarnessSubpath      = "harness"
-	HarnessProcessFile  = "process.pid"
-	HarnessLogFile      = "harness.log"
+	// Modelkits are stored in $KITOPS_HOME/storage/ and
+	// credentials are stored in $KITOPS_HOME/credentials.json
+	DefaultConfigSubdir               = "kitops"
+	StorageSubpath                    = "storage"
+	CredentialsSubpath                = "credentials.json"
+	HarnessSubpath                    = "harness"
+	HarnessProcessFile                = "process.pid"
+	HarnessLogFile                    = "harness.log"
+	UpdateNotificationsConfigFilename = "disable-update-notifications"
```

Comment:
Should we just have a `kitconfig.json` where this is just a key?

Reply:
That's a much bigger task :)

If we have a kitconfig.json, I'd expect to be able to almost configure everything there (e.g. something like viper). What that looks like is pretty nebulous.

Reply:
let's file an issue for it

Reply:
Filed https://github.com/jozu-ai/kitops/issues/419